auto_nag/team_managers.py

```diff
@@ -36,3 +40,81 @@ def get_team_manager(
                 return None
 
         return self.managers[team_name]
+
+    def _fetch_component_teams(self):
```

Comment:
Nit: don't forget to add type annotations to this and other functions

Reply:
Thank you for the remainder. The new functions with no annotations are void private functions that take no parameters. I added an annotation to an old function in e62c7f129917ce9d49486619667696076d5a5b06.

Reply:
They should be annotated anyway, otherwise mypy won't check them.
E.g. in this case it should be `def _fetch_component_teams(self) -> None:`

Reply:
Done in 34ac08649217a4486d808fd42b5f0e6c49f21de8